Sources/Purchasing/Purchases/Purchases.swift

```diff
@@ -260,6 +260,7 @@ public typealias StartPurchaseBlock = (@escaping PurchaseCompletedBlock) -> Void
     convenience init(apiKey: String,
                      appUserID: String?,
                      userDefaults: UserDefaults? = nil,
+                     documentsDirectory: URL? = nil,
```

Comment:
Optional for integration tests, similar to `userDefaults`, allows us to override with a temporary one that gets removed on each test run.